Remotion/Data/DomainObjects.UnitTests/Infrastructure/DomainObjectTransactionContextIndexerTest.cs

```diff
@@ -29,24 +30,13 @@ public void Item ()
     {
       var order = DomainObjectIDs.Order1.GetObject<Order>();
       var tx = order.RootTransaction.CreateSubTransaction();
-      var indexer = new DomainObjectTransactionContextIndexer(order, false);
+      var indexer = new DomainObjectTransactionContextIndexer(new DomainObjectTransactionContextImplementation(order));
```

Comment:
After reading this test for the third or so time, I realized we can solve this without the private invoke.

Step 1:
```suggestion
      var transactionContextImplementation = new DomainObjectTransactionContextImplementation(order)
      var indexer = new DomainObjectTransactionContextIndexer(transactionContextImplementation);
```